docs/source/isidora.rst

```diff
@@ -1,5 +1,54 @@
 Isidora
 ==========
 
-.. caution::
-  This page is currently not created, we are working on it. If you would like to contribute on this documentation, reach `me <helena.almamol@gmail.com>`_!
+*From ancient Greek, Ἰσίδωρος: gift of Isis.*
+
+Due to the bias configuration of the PMTs, the PMT waveform does not represent the actual signal produced by the PMT, but its derivative. This effect is treated online (for trigger) and offline (for data processing) by means of the BaseLine Restoration (BRL) algorithm. *Isidora* processes **RWF**\ s coming from either the *Decoder* (detector) or from *Diomira* (MC) and applies the BLR algorithm to the waveforms. The output are the so-called Corrected Waveforms, or **CWF**.
```

Comment:
I would add a line saying where these CWF are used, or if they can be the input of any city (not sure, in fact).

Reply:
No, CWFs are not used anywhere else. That's the reason for the note that follows. Any suggestions on how to make it better without repetition?

Reply:
You're right. I would leave it as it is. Maybe just add some bold in 'debugging' on the note. (Btw, I resolved the other comment I left on the review, don't forget to add it!)